### ISSUE TITLE
unnesary documentation for hosting!!

### ISSUE THREAD

**Issue by uknownnmonitor** [aws-amplify/docs#7855]
Amplify hosting services documentation for unnecessary platforms is misleading! like flutter, android, swift, angular js 

<img width="1403" alt="Screenshot 2024-07-28 at 9 24 55 PM" src="https://github.com/user-attachments/assets/dfb58641-09f0-4c8e-af13-79b266c1616b">

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by ironspur5** [comment]
Agreed, incorrect documentation here. I have only seen one reference of Amplify Hosting working with Flutter for example: https://medium.com/flutter-community/hosting-flutter-web-applications-with-amplify-hosting-and-github-actions-d53ba213767.

**Comment by uknownnmonitor** [comment]
you are right! @ironspur5, super clean examples in the documentation are super helpful ❤️ ! maybe we should paste this example in all the pages, thanks for pointing me in the right direction.
